src/export/bops/index.ts

```diff
@@ -503,7 +503,7 @@ export const extractFileDescriptionForPassportKey = (
 ): string | undefined => {
   try {
     // XXX: check for .description or .reason as there might be either atm
-    //      i.e. file = property.photograph, text = property.photograph.reason
+    //      i.e. file = photographs.existing, text = photographs.existing.reason
     for (const x of ["description", "reason"]) {
```

Comment:
these pre-existing suffixes on followup questions to existing single file uploads still work as expected with new naming patterns :+1:

am not accounting for them yet in the planx-data-migration script though - will check in with George & August to understand if these are still used in live content or have since been retired since we last updated our mocks

Reply:
now captured in data migration script here https://github.com/theopensystemslab/planx-data-migrations/pull/7